About/Manifest.xml

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <Manifest>
   <identifier>Hospitality</identifier>
-  <version>1.3.1.6</version>
+  <version>1.4</version>
```

Comment:
The version should be 1.4.0.1

Reply:
Added commit [c5b7efc](https://github.com/OrionFive/Hospitality/pull/703/commits/c5b7efc5ac35fe550807e995c4a9601c118bda04) to fix this Manifest and AssemblyInfo